packages/ui/src/common/components/buttons/LinkButtons.tsx

```diff
@@ -91,7 +91,7 @@ const LinkButtonInnerWrapper = styled.span<LinkButtonSizingProps>`
 `
 
 interface LinkButtonStyleProps extends Omit<LinkButtonProps, 'square'> {
-  $square: LinkButtonProps['square']
+  square?: LinkButtonProps['square']
 }
```

Comment:
The `$square` (from #1034) is to prevent `square` to be passed down to the Link component which causes: `Warning: Received true for a non-boolean attribute square` when using React dev tools (it doesn't append with `<ButtonXXX />`). I don't know styled-components very well, is there another way to prevent this warning ?

Reply:
Where can I find this error?

Reply:
I'm not sure where on the app, but you can see it in the `Common/Buttons/LinkButtons` story (it first needs to be wrapped in `<MemoryRouter />` though)